docs/product/dev-mode.md

```diff
@@ -0,0 +1,126 @@
+<!--
+title: Dev Mode
+menuText: Dev Mode
+description: A guide for using logs within Serverless Console
+menuOrder: 5
+-->
+
+# Development Mode
```

Comment:
This may or may not make a difference, but I'd keep the header as "Dev Mode", with the hope that it helps SEO.

Reply:
Might as well.